coderd/coderdtest/coderdtest.go

```diff
@@ -725,6 +727,79 @@ func NewAWSInstanceIdentity(t *testing.T, instanceID string) (awsidentity.Certif
 		}
 }
 
+type FakeOIDCConfig struct {
+	key    *rsa.PrivateKey
+	issuer string
+}
+
+func NewFakeOIDCConfig(t *testing.T, issuer string) *FakeOIDCConfig {
```

Comment:
I might just call this `OIDCConfig` instead. Not a biggie and totally a nit tho. Since it's in the `coderdtest` pkg it might be explicit

Reply:
I actually called it that before I renamed it, I'll revert it.